pkg/adapter/mtping/adapter_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	sourcesv1beta1 "knative.dev/eventing/pkg/apis/sources/v1beta1"
+	sourcesv1beta2 "knative.dev/eventing/pkg/apis/sources/v1beta2"
```

Comment:
Would be nice to have a consistent naming: `sourcesv1beta2` is used here and `v1beta2` is used in `adapter.go` (above). My preference is for `v1beta2` without the alias (since the package already has that name).

Reply:
thanks, done